Build failure on `android/termux`

```
[github.com/projectdiscovery/naabu/v2/pkg/result](http://github.com/projectdiscovery/naabu/v2/pkg/result)
# [github.com/projectdiscovery/utils/permission](http://github.com/projectdiscovery/utils/permission)
go/pkg/mod/github.com/projectdiscovery/utils@v0.0.60/permission/permission_other.go:10:6: checkCurrentUserRoot redeclared in this block
        go/pkg/mod/github.com/projectdiscovery/utils@v0.0.60/permission/permission_linux.go:16:6: other declaration of checkCurrentUserRoot
go/pkg/mod/github.com/projectdiscovery/utils@v0.0.60/permission/permission_other.go:15:6: checkCurrentUserCapNetRaw redeclared in this block
        go/pkg/mod/github.com/projectdiscovery/utils@v0.0.60/permission/permission_linux.go:21:6: other declaration of checkCurrentUserCapNetRaw
[github.com/projectdiscovery/retryabledns/doh](http://github.com/projectdiscovery/retryabledns/doh)
```
ref: https://github.com/projectdiscovery/httpx/pull/1424

--- a/v2/go.mod
+++ b/v2/go.mod
@@ -22,7 +22,7 @@ require (
 	github.com/projectdiscovery/ratelimit v0.0.13
 	github.com/projectdiscovery/retryablehttp-go v1.0.33
 	github.com/projectdiscovery/uncover v1.0.7
-	github.com/projectdiscovery/utils v0.0.60
+	github.com/projectdiscovery/utils v0.0.61
 	github.com/remeh/sizedwaitgroup v1.0.0
 	github.com/stretchr/testify v1.8.4
 	go.uber.org/multierr v1.11.0
@@ -65,7 +65,7 @@ require (
 	github.com/mattn/go-runewidth v0.0.14 // indirect
 	github.com/mholt/archiver v3.1.1+incompatible // indirect
 	github.com/microcosm-cc/bluemonday v1.0.25 // indirect
-	github.com/minio/selfupdate v0.6.0 // indirect
+	github.com/minio/selfupdate v0.6.1-0.20230907112617-f11e74f84ca7 // indirect
 	github.com/modern-go/concurrent v0.0.0-20180306012644-bacd9c7ef1dd // indirect
 	github.com/modern-go/reflect2 v1.0.2 // indirect
 	github.com/muesli/reflow v0.3.0 // indirect

